src/UI/Components/Diagram/actions.ts

```diff
@@ -206,56 +237,125 @@ export function appendInstance(
   if (!serviceInstanceModel) {
     throw Error(words("instanceComposer.errorMessage"));
   }
-  const instanceAsTable = new ServiceEntityBlock().setName(
-    serviceInstance.service_entity,
+
+  const attributes =
+    serviceInstance.candidate_attributes ||
+    serviceInstance.active_attributes ||
+    undefined;
```

Comment:
Since start candidate had priority over candidate, since these are the ones that would be edited as far as I remember.

I can't find the file you mentioned, I think I used undefined in every assignments of the composer entities